assets/templates/signup.html

```diff
@@ -149,11 +149,11 @@ <h5>email, automation & data marketing</h5>
           <small>© 2019 Doppler LLC. Todos los derechos reservados. <a href="#" target="_blank">Política de Privacidad y Legales.</a></small>
         </footer><!-- footer -->
       </article>
-      <section class="feature-panel">
+      <section class="feature-panel bg--signup">
         <article class="feature-content">
-          <h6>Editor de emails</h6>
-          <h3>Crea Emails en minutos y accede a nuestra Galería de Plantillas</h3>
-          <p>Nuestras Plantillas para Email son totalmente Responsive y fácilmente editables desde nuestro Editor HTML.</p>
+          <h6>email automation</h6>
```

Comment:
Same here very minor `Email Automation` The caps on `Automation` is because all doppler items like Lists and Campaigns are wirtten this way everywhere in the app.